src/core/operations_problem.jl

```diff
@@ -530,8 +537,8 @@ function solve!(
 )
     solve_status = solve!(problem)
     if solve_status == RunStatus.SUCCESSFUL
-        stats = OptimizerStats(problem, step, start_time)
-        append_optimizer_stats!(store, stats)
+        stats = OptimizerStats(problem, step)
+        write_optimizer_stats!(store, get_name(problem), stats)
```

Comment:
Note that the stats are not exported during the simulation. Let me know if that is needed.

Reply:
unlikely. that's more of an after the fact metric